code/src/App.js

```diff
@@ -1,9 +1,330 @@
-import React from 'react'
+import React, { useState } from 'react'
+import { Summary } from './Summary'
+import "./app.css";
+import './progressbar.css';
```

Comment:
Decide if you want the project to have `"` or `'` and stick to it

Reply:
Hej Erica,
 Tack! Angående aria-label, kan du ge ett exempel hur/var jag ska använda det? Jag har använt aria-label men isf på fel sätt? Jag har provat att ändra till aria-label="radio-button"> men det funkar ändå inte. 

{question === 6 && (
            <div className="question">
              <div className="feelings">
                <h3>Are you happy with that amount?</h3>
                <div className="emojis">
                  <label>
                    <input type="radio"
                      value="happy"
                      onChange={() => setHappiness("happy")}
                      checked={happiness === "happy"}
                    />
                    **<span role="img" aria-label="happy face">**
                      😅
                </span>
                  </label>
                  <label>
                    <input type="radio"
                      value="sad"
                      onChange={() => setHappiness("sad")}
                      checked={happiness === "sad"}
                    />
                    **<span role="img" aria-label="sad sweaty face">**
                      😓
                </span>
                  </label>
                </div>
              </div>
            </div>
          )}